src/components/JoinUs/Work.js

```diff
@@ -1,13 +1,9 @@
 import React from 'react'
-import breakpoint from 'styled-components-breakpoint'
-import remcalc from 'remcalc'
-import { Link } from 'gatsby'
+
 import { Padding } from 'styled-components-spacing'
-import styled from 'styled-components'
 import { ColumnLayout } from '../grid'
-import { CardTitle, BodyPrimary } from '../Typography'
 import colorLuminance from 'color-luminance'
-
+import { CaseStudyWrapper, CaseStudy } from '../Common/CaseStudy'
```

Comment:
So here lets remove `CaseStudyWrapper` and just create the padding on the `<Col>` or the `CaseStudy` itself.

Reply:
Left the wrapper because it does flexing but moved it into styled component